src/utils_format_json.c

```diff
@@ -484,4 +495,215 @@ int format_json_value_list (char *buffer, /* {{{ */
         store_rates, (*ret_buffer_free) - 2));
 } /* }}} int format_json_value_list */
 
+#if HAVE_LIBYAJL
+static int json_add_string (yajl_gen g, char const *str) /* {{{ */
```

Comment:
Minor nit: const char *
I don't mind personally but I noticed this confuses less experienced contributors who think the order does matter.